main.py

```
@@ -14,6 +14,8 @@
     ApplicationBuilder, CommandHandler, MessageHandler, ContextTypes, filters
 )
 import google.generativeai as genai
+# Import glm for embedding functionalities
+import google.generativeai.types as glm
```

Ruff (0.11.9)

`google.generativeai.types` imported but unused

Remove unused import: `google.generativeai.types`

(F401)